clusterloader2/testing/density/config.yaml

```diff
@@ -1,11 +1,19 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$NODES_PER_NAMESPACE := 100}}
-{{$PODS_PER_NODE := 30}}
+{{$PODS_PER_NAMESPACE := 3000}}
+{{$MIN_LATENCY_PODS := 500}}
+{{$MIN_SATURATION_PODS_TIMEOUT_S := 120}}
+{{$POD_THROUGHPUT_PER_S := 20}}
 #Variables
-{{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
-{{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
-{{$replicas := DivideInt $totalPods $namespaces}}
-{{$totalPodsRounded := MultiplyInt $namespaces $replicas}}
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+{{$totalPods := MultiplyInt $PODS_PER_NAMESPACE $namespaces}}
+{{$latencyReplicas := DivideInt (MaxInt $MIN_LATENCY_PODS .Nodes) $namespaces}}
+{{$totalLatencyPods := MultiplyInt $namespaces $latencyReplicas}}
+{{$saturationRCTimeout := MaxInt $MIN_SATURATION_PODS_TIMEOUT_S (DivideFloat $totalPods $POD_THROUGHPUT_PER_S)}}
```

Comment:
That is too short:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/density.go#L637